src/app/queries/mocks/storages.mock.ts

```diff
@@ -1,8 +1,66 @@
-import { IAnnotatedStorageClass, IByProvider } from '../types';
+import { ISourceStorage, IAnnotatedStorageClass, IByProvider } from '../types';
 
+export let MOCK_VMWARE_DATASTORES: ISourceStorage[] = [];
+export let MOCK_RHV_STORAGE_DOMAINS: ISourceStorage[] = [];
 export let MOCK_STORAGE_CLASSES_BY_PROVIDER: IByProvider<IAnnotatedStorageClass> = {};
 
 if (process.env.NODE_ENV === 'test' || process.env.DATA_SOURCE === 'mock') {
+  MOCK_VMWARE_DATASTORES = [
+    {
+      id: '1',
+      name: 'vmware-datastore-1',
+      selfLink: '/foo/vmwaredatastore/1',
+    },
+    {
+      id: '2',
+      name: 'vmware-datastore-2',
+      selfLink: '/foo/vmwaredatastore/2',
+    },
+    {
+      id: '3',
+      name: 'vmware-datastore-3',
+      selfLink: '/foo/vmwaredatastore/3',
+    },
+    {
+      id: '4',
+      name: 'vmware-datastore-4',
+      selfLink: '/foo/vmwaredatastore/4',
+    },
+    {
+      id: '5',
+      name: 'vmware-datastore-5',
+      selfLink: '/foo/vmwaredatastore/5',
+    },
+  ];
```

Comment:
It's for the same reason. Jeff requested that we pare down these to only what we need, so that he has the option of making the objects more sparse in the future to improve performance (or changing the fields we're not using without possibly breaking us in the future if we decide to use them).

Reply:
I have an issue open to do this same kind of reduction in types for everything else. https://github.com/konveyor/forklift-ui/issues/618